build/build/src/ide/web/dejavu_font.rs

```diff
@@ -0,0 +1,173 @@
+//! Downloading DejaVu Fonts.
+
+use crate::prelude::*;
+
+use enso_font as font;
+use enso_font::NonVariableDefinition;
+use enso_font::NonVariableFaceHeader;
+use ide_ci::cache::Cache;
+
+
+
+// =================
+// === Constants ===
+// =================
+
+pub const PACKAGE_URL: &str = "https://github.com/dejavu-fonts/dejavu-fonts/releases/download/version_2_37/dejavu-fonts-ttf-2.37.zip";
+
+const PACKAGE_FONTS_PREFIX: &str = "DejaVu";
+
+const PACKAGE_SANS_MONO_PREFIX: &str = "SansMono";
+
+const DEJAVU_SANS_MONO_FONT_FAMILY_FONTS: &[(&str, font::Weight)] =
+    &[("-Bold", font::Weight::Bold), ("", font::Weight::Normal)];
+
+
+
+// ===================
+// === DejaVu Font ===
+// ===================
+
+pub async fn download_dejavu_sans_mono_font_internal(
```

Comment:
This function seems to largely duplicate `install_enso_font_for_html_2`. Parts not specific to this font (deja vu sans mono) should be split away and reused.